system/greenhouse-ccloud/templates/thanos-pluginpreset.yaml

```diff
@@ -25,10 +25,12 @@ spec:
     displayName: Thanos
     pluginDefinition: thanos
     releaseNamespace: kube-monitoring
-    {{- if and .Values.global.quayIoMirror .Values.global.ghcrIoMirror }}
+    {{- if and .Values.global.quayIoMirror .Values.global.ghcrIoMirror .Values.global.dockerHubMirror }}
     optionValues:
     - name: thanos.image.repository
       value: {{ printf "%s/thanos/thanos" .Values.global.quayIoMirror }}
+    - name: thanos.initChownData.image.registry
+      value: {{ .Values.global.dockerHubMirror }}
```

Comment:
Is this particular image pinned to dockerHub? Just by looking at it I'd say it could use the value in `thanos.image.repository`.

Reply:
Yes.

No, because `thanos.image.repository` contains of `registry` and `repository`. And the registry is `quay.io`. 

 This PR is related to this change: https://github.com/cloudoperators/greenhouse-extensions/pull/614